shell/platform/fuchsia/runtime/dart/profiler_symbols/dart_profiler_symbols.dart

```diff
@@ -82,9 +80,11 @@ class Symbol {
   int offset;
   int size;
   String name;
+
+  Symbol(this.offset, this.size, this.name);
```

Comment:
nit: for readability at the call site, maybe do

```dart
Symbol({required this.offset, required this.size, required this.name});
```

and then call it as:

```dart
final symbol = Symbol(offset: int.parse(match[1]!, radix: 16),
                      size: int.parse(match[2]!, radix: 16),
                      name: match[4]!.split("(")[0]);
```

so that the reader can tell what each value represents at the call site. 

(the `new` keyword is unnecessary, it does the same thing without it in Dart 2).

Reply:
Thanks. I'm not experienced with dart so any tips like this are vey helpful!